R/measures.R

```diff
@@ -64,8 +68,13 @@
 #' }
 riem_measures <- function(
     station = "VOHY",
-    date_start = "2014-01-01",
```

Comment:
I wonder if instead of updating it we should remove the default, same for station. So the idea would be

- no default for station
- no default for date_start

It's a breaking change but I think it's a worthwhile one.

Then we could add an ellipsis meant to be empty to ensure users name optional arguments.
Reference: https://design.tidyverse.org/dots-after-required.html


```r
 @inheritParams rlang::args_dots_empty
riem_measures <- function(
    station = "VOHY",
    date_start = "2014-01-01",
   ...,
    date_end = as.character(Sys.Date()),
    elev = FALSE,
    latlon = TRUE,
    report_type = "3,4") {

  rlang::   check_dots_empty()
```

Maybe even `date_end` should not have a default but I'm less sure about that.

Reply:
I think this is a great plan.  I'll make it so.  This is what I wanted to do in the first place, but feared making breaking changes.

I'm not sure if this project employs [semantic versioning](https://semver.org/), but would this change constitute a need to bump a major version number?

As for `date_end`, I personally find the default of today's date to be helpful.

Reply:
> I'm not sure if this project employs [semantic versioning](https://semver.org/), but would this change constitute a need to bump a major version number?

Yes, that's a good idea. I can make a new release after the PR is merged (in the new year). Please list yourself as "aut" in `DESCRIPTION` as your PR is a substantial contribution.

Ok for `date_end`! Truth be told, I don't use riem myself these days. :grin: